index.js

```diff
@@ -42,17 +44,19 @@ function processArgv(argv) {
 
   switch (command) {
     case 'version':
+      verbose = true;
```

Comment:
So we are always defaulting to verbose being true for version and undefined? I like it.

Reply:
Yeah.  `version` and `help`/`undefined` seemed like ones to always log.  Especially based on you introducing answering the `version` command.  :-)